blog/docs/events/virtual-office-hour-12-2022.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "First 🌱 Knative Virtual Office Hour / Contrib Fest 2022/12/09"
+type: "blog"
+date: 2022-12-01
+linkTitle: "First Knative Virtual Office Hour / Contrib Fest 2022/12/09"
+description: "Join the Zoom Knative First Knative Virtual Office Hour / Contrib Fest on 2022/12/09"
```

Comment:
I might call this "Virtual Office Hour" and drop the "Contrib Fest" part. I think having "open for new contributors" in the text body is okay, but I think having end-users show up just to ask questions or chat should be explicitly our goal, and adding "ContribFest" adds a bunch of pressure to become a contributor.